.github/workflows/ci.yml

```diff
@@ -43,6 +43,11 @@ jobs:
         run: npm run build
       - name: Lint
         run: npm run is-it-pretty
+      - name: Unset git config safe.directory globally
+        run: git config --global --unset-all safe.directory || true
+      - name: Unset git config safe.directory for linux
+        if: runner.os == 'Linux'
+        run: sudo git config --system --unset-all safe.directory || true
```

Comment:
Linux sets safe.directory=* at the --system config. I tried simply unsetting it but got "can't lock file" error. When googling seems sudo is what you have to do to modify the --system config.. But if there is a better way I always open. :/